<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nodo</class>
 <widget class="QWidget" name="Nodo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos del sensor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/sigrdap/icons/nuevo.png</normaloff>:/sigrdap/icons/nuevo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
background-color: transparent;
border-style: solid;
border-bottom: 2px solid #3498db;}

QLabel{
color: rgb(154, 154, 154)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLineEdit" name="editIdDispositivo">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Identificador del dispositivo</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Identificador del dispositivo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editCalle">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Calle</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Calle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editColonia">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Colonia</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Colonia</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editCP">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>CP</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>CP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Municipio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selectorMunicipio">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Municipio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Grupo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selectorGrupo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Grupo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Tipo de sensor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="selectorTipoSensor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Tipo de sensor</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="editNivelMaximo">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Máximo alarma</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Máximo alarma</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDerMax">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="editNivelMinimo">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Mínimo alarma</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Mínimo alarma</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDerMin">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="editArea">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Área</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Área</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDerArea">
           <property name="text">
            <string>m²</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="editAltura">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Altura</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Altura</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDerAlt">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="editGrafica">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Máximo gráfica</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Máximo gráfica</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDerGrafica">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="activarAlarma">
         <property name="text">
          <string>Activar alarma</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/VentanaZigBee/icons/switch.png</normaloff>:/VentanaZigBee/icons/switch.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCommandLinkButton" name="botonOnline">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>En línea</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/VentanaZigBee/icons/broadcast.png</normaloff>:/VentanaZigBee/icons/broadcast.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonGuardar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:enabled{
background-color: #3498db;
color: rgb(255, 255, 255);
}

QPushButton:disabled{
background-color: #adc1ce;
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>GUARDAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editIdDispositivo</tabstop>
  <tabstop>editCalle</tabstop>
  <tabstop>editColonia</tabstop>
  <tabstop>editCP</tabstop>
  <tabstop>selectorMunicipio</tabstop>
  <tabstop>selectorGrupo</tabstop>
  <tabstop>selectorTipoSensor</tabstop>
  <tabstop>editNivelMaximo</tabstop>
  <tabstop>editNivelMinimo</tabstop>
  <tabstop>editArea</tabstop>
  <tabstop>editAltura</tabstop>
  <tabstop>editGrafica</tabstop>
  <tabstop>activarAlarma</tabstop>
  <tabstop>botonOnline</tabstop>
  <tabstop>botonGuardar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
